src/EnergyPlus/SetPointManager.cc

```diff
@@ -5312,10 +5320,19 @@ namespace SetPointManager {
 		int FanInNode; // supply fan inlet node number
 		int FanOutNode; // supply fan outlet node number
 		int RefNode; // setpoint reference node number
+		int CoolCoilInNode; // Cooling coil inlet node number
+		int CoolCoilOutNode; // Cooling coil outlet node number
+		Real64 MinTemp; // Minimum temperature at cooling coil outlet node
+		Real64 dtFan; // Temperature difference across a fan
+		Real64 dtCoolCoil; // Temperature difference across a coolig coil
 
 		FanInNode = this->FanInNode;
 		FanOutNode = this->FanOutNode;
 		RefNode = this->RefNode;
+		CoolCoilInNode = this->CoolCoilInNode;
```

Comment:
It's not great, but if it is functionally performing properly (no shadowing antics), then I don't think we should hold it up with today's freeze.  We could potentially note it as a clean-up to do post-freeze...